rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1289,6 +1289,12 @@ void main_window::EnableMenus(bool enabled)
 	ui->toolsRsxDebuggerAct->setEnabled(enabled);
 	ui->toolsStringSearchAct->setEnabled(enabled);
 	ui->actionCreate_RSX_Capture->setEnabled(enabled);
+
+	if (!enabled)
+	{
+		if (m_kernel_explorer)
```

Comment:
Why here????
Just put it to the OnEmuStop function

Reply:
Oh, okay. Fixed.